GenumKit/Parsers/ColorsFileParser.swift

```diff
@@ -66,3 +73,47 @@ public final class ColorsFileParser {
     return value
   }
 }
+
+//MARK: - CLR File Parser
+
+public final class CLRFileParser: ColorsFileParser {
+  public private(set) var colors = [String: UInt32]()
+  
+  public init() {}
+  
+  public func parseFile(path: String) {
+    if let colorsList = NSColorList(name: "UserColors", fromFile: path) {
+      for keyColorPair in colorsList.allKeys.map({ ($0, colorsList.colorWithKey($0)) }) {
+        let colorName = keyColorPair.0.stringByReplacingOccurrencesOfString(" ", withString: "")
+        if let colorValue = keyColorPair.1?.rgbColor?.hexValue {
+          addColorWithName(colorName, value: colorValue)
+        }
+      }
+    }
+  }
+  
+  public func addColorWithName(name: String, value: String) {
+    addColorWithName(name, value: ColorsTextFileParser.parse(value))
+  }
+  
+  public func addColorWithName(name: String, value: UInt32) {
+    colors[name] = value
+  }
+
+}
+
+extension NSColor {
+  
+  private var rgbColor: NSColor? {
+    return colorUsingColorSpaceName(NSCalibratedRGBColorSpace)
+  }
+  
+  private var hexValue: UInt32 {
+    let hexRed = ((UInt32(redComponent * 255.0) << 8) << 8) << 8
+    let hexGreen = (UInt32(greenComponent * 255.0) << 8) << 8
+    let hexBlue = UInt32(blueComponent * 255.0) << 8
+    let hexAlpha = UInt32(alphaComponent * 255.0)
+    return hexRed + hexGreen + hexBlue + hexAlpha
```

Comment:
You should use `|` here (bitwise-OR) and not `+` (arithmetic addition) to combine the different bytes together.